sky/backends/cloud_vm_ray_backend.py

```diff
@@ -1035,11 +1036,56 @@ def _retry_region_zones(self,
                    'Try changing resource requirements or use another cloud.')
         raise exceptions.ResourcesUnavailableError(message)
 
+    def tpu_pod_setup(self, cluster_yaml: str,
```

Comment:
It seems this function is only used in this file. Should we fix the name to `_tpu_pod_setup`?

Reply:
Updated!